### ISSUE TITLE
The recovery server could not be contacted but has internet access

### ISSUE THREAD

**Issue by acheong08** [sickcodes/Docker-OSX#543]
Network is working, I can ping etc but recovery server still cannot be contacted. I have a Macbook that has ran into the same issue in the past month so I'm not sure if it a Docker-OSX issue or Apple issue

**Comment by acheong08** [comment]
It might be a DNS issue

**Comment by acheong08** [comment]
Damn I hate China

**Comment by DrStoop** [comment]
I got the same messages while installing the image sickcodes/docker-osx:ventura and simply clicked them away and checked the internet in the Safari browser. As long as my System Monitor was showing continuous downloading and increased CPU activity I simply let it run. I also had other error messages which I simply clicked away and worst case restarted the installation. After the download was complete multiple reboots were required to finish the installation off. The installation took some hours (probably mainly due to slow internet) but did finish.
